src/details/ArborX_DetailsVector.hpp

```diff
@@ -75,13 +75,18 @@ Vector(T, U...) -> Vector<1 + sizeof...(U), T>;
 
 } // namespace ArborX::Details
 
-template <typename Point, typename Enable = std::enable_if_t<
-                              ArborX::GeometryTraits::is_point_v<Point>>>
+// FIXME: remove second template argument when ExperimentalHyperGeometry switch
+// happens
+template <typename Point, typename Point2,
```

Comment:
We do this because some of the ray geometry algorithms currently mix two point types, as they are not templated on the point type. I didn't want to convert multiple places there, preferring to do this small change.

Reply:
OK